jdatetime/__init__.py

```diff
@@ -41,6 +40,13 @@
 else:
     FA_LOCALE = 'fa_IR'
 
+# Making translators
+number_converter = dict((ord(a), b) for a, b in zip(
```

Comment:
I think the `number_converter` is missing 3 characters from the specified list of unicode characters:
```
>>> from jdatetime import number_converter
>>> len(number_converter.keys())
66
>>> numerics = u'١٢٣٤٥٦٧٨٩٠۱۲۳۴۵۶۷۸۹۰٤٥٦₀₁₂₃₄₅₆₇₈₉¹²⁰⁴⁵⁶⁷⁸⁹①②③④⑤⑥⑦⑧⑨⑴⑵⑶⑷⑸⑹⑺⑻⑼⒈⒉⒊⒋⒌⒍⒎⒏⒐'
>>> len(numerics)
69
```

We can find the duplicates:
```
>>> ords = [ord(num) for num in numerics]
>>> len(ords)
69
>>> from collections import Counter
>>> [item for item, count in Counter(ords).items() if count > 1]
[1636, 1637, 1638]
```

So 3 of the items are mapped to the same key.

Python version info:
```
Python 2.7.12 (default, Nov 12 2018, 14:36:49) 
[GCC 5.4.0 20160609] on linux2
```

Reply:
Fixed, it was a result from copy-pasting without double checking. Thanks